app/controllers/planning_applications_controller.rb

```diff
@@ -26,24 +26,53 @@ def show
   def edit
   end
 
-  # rubocop:disable Metrics/MethodLength
   def update
```

Comment:
If this method isn't doing anything, and isn't being used, can we delete it and remove it from the routes?